sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/BasicRelationship.cs

```diff
@@ -13,6 +13,43 @@ namespace Azure.DigitalTwins.Core.Serialization
     /// <remarks>
     /// For more samples, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples">our repo samples</see>.
     /// </remarks>
+    /// <example>
+    /// Here's an example of  how to use the BasicRelationship helper class to serialize and create a relationship.
```

Comment:
Might add a link to the model the twins adhere to, so the user can reference them.

Reply:
The model that this twin adheres to is a temporary model created in the SamplesConstant. Since it's a different namespace, I don't think I can do a '<see cref..' into a different namespace. That is assuming I am not misunderstanding your comment :)

Reply:
Will merge the PR and discuss with you this comment tomorrow. I can make a separate PR.

Reply:
Yes, you can do a different namespace, but I was thinking of using href instead of cref and pointing to the file on github.